packages/react-components/package.json

```diff
@@ -62,6 +62,7 @@
     "uuid": "^9.0.0"
   },
   "peerDependencies": {
+    "@azure/communication-calling": "1.23.1-beta.2 || ^1.22.1",
```

Comment:
This package is intentionally agnostic of ACS, instead recreate types. Some type interfaces will look the same, others will need to be reshaped to be an agnostic UI prop.
Looks like this is just for "Reaction" which can recreate as appropriate

Reply:
We already have a `Reaction` type in the `react-component`. Having another would create conflict. So importing from the original calling lib. Can I have some pointer how to resolve this?

Reply:
the expected signature for both types are different.

Reply:
The parent using the UI component then has to map whatever the component type is to its function, e.g.
```tsx
const myFunction = (varStr: 'string') => { console.log(varStr) };

interface ComponentProps: {
  doSomething: (varNum: number) => void;
}

<Component
  // do something requires number, but myFunction want a string, hence we need to convert the
  // arg from doSomething to match the expected input to myFunction, in this case a numToStr function
  doSomething={(varNum) => myFunction(numToStr(varNum))}
/>

```

Reply:
Thanks. got it. Pushed a new commit addressing the concern.